index.html

```diff
@@ -0,0 +1,499 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>laslessVPN</title>
+    <link rel="stylesheet" href="./styles/style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <img
+          src="./img/logo.svg"
+          width="149"
+          height="36"
+          alt="lasslessvpn-logo"
+        />
+        <ul class="header__menu">
+          <li class="header__item">
+            <a class="header__link" href="#">About</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#">Features</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#">Pricing</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#">Testimonials</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#">Help</a>
+          </li>
+        </ul>
+        <div class="container__button">
+          <button class="button button__sign-in" type="button">Sign in</button>
+          <button class="button button__sign-up" type="button">Sign up</button>
+        </div>
+      </div>
+    </header>
+    <main class="main">
+      <section class="main-screen">
+        <h1 class="main-screen__title">
+          Want anything to be easy with
+          <strong class="title--accent">LaslesVPN.</strong>
+        </h1>
+        <p class="main-screen__text">
+          Provide a network for all your needs with ease and fun using
+          <strong class="text--accent">LaslesVPN</strong> discover interesting
+          features from us.
+        </p>
+        <button class="button main-screen__button" type="button">
+          Get Started
+        </button>
+      </section>
+
+      <section class="advantages">
+        <div class="advantages__wrap">
+          <div class="advantages__wrap--item">
```

Comment:
According to BEM, `--item` is a modifier which cannot be used without general class. You don't seem to need here `wrap` word at all, why don't you use simply `advantages__item`?

Reply:
+